portfolio/src/main/webapp/index.html

```diff
@@ -3,34 +3,45 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title> Carol's Portfolio </title>
+    <title>Carol's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div class="container">
-      <div class="menu">
-        <ul>
-          <li class="logo"> <a href="index.html"> <img src="images/logo.jpg"> </a> </li>
-          <li class="currentLink"> <a href="index.html" style="color:white"> About Me </a> </li>
-          <li> <a href="background.html"> Background </a> </li>
-          <li> <a href="photos.html"> Photos </a> </li>
-          <li> <a href="contact.html"> Contact Me </a> </li>
-        </ul>
-      </div>
+
+  <body onload=loadComments()>
+    <div class="menu">
+      <ul>
+        <li class="logo"><a href="index.html"><img src="images/logo.jpg"></a></li>
+        <li class="currentLink"><a href="index.html" style="color:white">About Me</a></li>
+        <li><a href="background.html">Background</a></li>
+        <li><a href="photos.html">Photos</a></li>
+        <li><a href="contact.html">Contact Me</a></li>
+      </ul>
     </div>
+
     <div class="banner">
-        <h1> About Me </h1>
-          <p> Hi! I'm Carol Sun! Here is a picture of me. </p>
-          <img src="images/Profile Pic.jpg" alt="Me" height="300" class="center"/>
-          <p> I am currently a STEP intern and a rising junior at Caltech. I live in the 
-          Los Angeles area and I like to cook and garden in my free time! </p>
-          <p style="text-align:center"> Click here to get a random fact!: </p>
-          <div class="button_wrapper">
-            <button onclick="addRandomFact()" class="button"> Fun Facts! </button> 
-          </div>
-          <br>
-          <div id="fact-container" class="center"> </div>
+      <h1>About Me</h1>
+      <p>Hi! I'm Carol Sun! Here is a picture of me. 
+      <img src="images/Profile Pic.jpg" alt="Me" height="300" class="center"/>
+      <p>I am currently a STEP intern and a rising junior at Caltech. 
+        I live in the Los Angeles area and I like to cook and garden in my free time! 
+      <p>Click here to get a random fact!: 
+      <div class="button_wrapper">
+        <button onclick="addRandomFact()" class="button">Fun Facts!</button> 
+      </div>
+      <br>
+      <div id="fact-container" class="center"></div>
+
+      <h2>Comments Section</h2>
+      <form method="POST" action="/data">
+		<textarea rows="4" cols="50" name="text">Leave a comment!</textarea>
+        <br>
+        <button>Submit</button>
+      </form>
+      <p>
```

Comment:
I was following the google style guide for HTML (https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags)
where they say to omit optional tags. I looked it up and apparently, `</p>` is optional so I deleted those tags from the code

Reply:
Another thing I noticed from the style guild is "For consistency and simplicity reasons it’s best served omitting all optional tags, not just a selection." In our case, for consistency it might be simpler to keep the optional tags in my opinion.